docs/OPENTELEMETRY.md

```diff
@@ -2,10 +2,10 @@
 
 > :warning: **Experimental**: opentelemetry is an evolving product, and so is our implementation of it.
 
-``diracx`` is capable of sending [OpenTelemetry](https://opentelemetry.io/) data to a collector. The settings are controled by the
-``diracx.routers.otel.OTELSettings`` classes
+`diracx` is capable of sending [OpenTelemetry](https://opentelemetry.io/) data to a collector. The settings are controled by the
```

Comment:
"controlled" not "controled" and shouldn't have this be spotted?

Reply:
Sure it did:
docs/CONFIGURATION.md:29: forsee ==> foresee
docs/DATABASES.md:18: vairable ==> variable
docs/OPENTELEMETRY.md:5: controled ==> controlled

But the change is not automatic, we have to fix it ourselves as with other precommit stuff....

Reply:
To be precise: I don't correct anything spotted by the linters just yet. I'm trying to eliminate false-positives, like reporting errors in generated code.